README.md

```diff
@@ -5,23 +5,36 @@ Install depedndencies
 ```
 npm install 
 ```
+
+### Before Running tests browser-sync needs to be started:
+```
+"start:browser-sync:prod": "NODE_ENV=prod node bs-config.js"
+"start:browser-sync:dev": "NODE_ENV=dev node bs-config.js"
+"start:browser-sync:stg": "NODE_ENV=stg node bs-config.js"
+```
 ### runs specific test
 ```
 npx codeceptjs run ./tests/<test_file_name>.js
 
 npx codeceptjs run ./tests/css_check_test.js
 ```
 
-### runs whole project
+### To run on certain env use:
 ```
-npx codeceptjs run 
-
-npm test
+"test": "npx codeceptjs run",
+"test:prod": "NODE_ENV=prod npx codeceptjs run",
+"test:dev": "NODE_ENV=dev npx codeceptjs run",
+"test:stg": "NODE_ENV=stg npx codeceptjs run"
 ```
 
 ### Configuration for project
-.env
-```
-HOST_URL - url where project should run
-SHOW_BROWSER - run in headless mode (`false` for headless mode)
+create separate ```.env``` files for different environments ```.env.dev .env.prod .env.stg``` etc. 
+.env 
+```
+HOST_URL
+SHOW_BROWSER
+WEBLOADER_URL
+PARTICIPANT_URL
+ENVIRONMENT_ID
+UID
```

Comment:
Looks like you need to explicitly add the _.env*_ files to the commit. _.env_ is often ignored by _.gitignore_ by default (which is true for this repo).

Reply:
As a possible alternative, in many of our other repos we use [node-config](https://www.npmjs.com/package/config) for configurations. I find it to be a bit more robust and flexible than `dotenv`.

Reply:
I didn't commit env files for security reasons. If you think it's ok to commit them, I'll add them to the repo

Reply:
I see. That's a good practice then.

How do you plan to include the environment variables when the tests are run on CI/CD?

Reply:
I believe most of ci\cd tools provide env variables setup. 
Github Actions are also provide this functionality but I haven't had my hands on that. https://docs.github.com/en/actions/reference/environment-variables

Reply:
This is where I think `node-config` is helpful. You can set up configuration files per environment that only include non-secret values that are checked in. And then using env vars set in Actions, you can inject secret values into the configuration.

Reply:
The repo has an example of how we use node-config:
https://github.com/SentientTechnologies/integrations.evolv.ai/tree/master/config

Reply:
For uniformity among our repos and for the extra functionality it brings, I think we should incorporate `node-config`.  Let me know if you need any guidance with this.

Reply:
I'll add `node-cofig` to this PR tomorrow morning with all configs

Reply:
This section is no longer relevant. You can reuse the language we use in other repos:
```markdown
To create a new environment configuration, copy the _config/default.js_ file into a new file in the same directory with the name _<environment>.js_.

> This repo leverages `node-config` for configuration files. See
> [here](https://github.com/lorenwest/node-config/wiki) for documentation about `node-config`.
> To make a local-only configuration, prefix the file name with _local-_.
> Any file matching `local-*` in the _config_ directory will be git-ignored, but make
> sure to not forcibly add the file to Git tracking.
```